pages.fr/common/kosmorro.md

```diff
@@ -0,0 +1,21 @@
+# kosmorro
+
+> Calcule les éphémérides et les événements pour une date donnée, à un emplacement donné sur Terre.
+> Plus d'informations : http://kosmorro.space.
+
+- Obtenir les éphémérides pour Paris (France) :
```

Comment:
This one has France in parentheses, while the common one doesn't. I think we should follow the same pattern as the common page. Or unless I have missed some specific French grammar.

Reply:
It's actually more common in French to put the country in parentheses rather than separated with a coma :)